packages/chat-client/src/controllers/engine.ts

```diff
@@ -745,6 +778,12 @@ export class ChatEngine extends IChatEngine {
 
       if (!decodedPayload) throw new Error("Empty ID Auth payload");
 
+      if (decodedPayload.act !== "invite_approval") {
+        this.client.logger.error(
```

Comment:
Same q as above, regarding this being handled but apparently not aborting the flow.

Reply:
Good catch, added an early `return`, forgot to copy it over from the handling for `chat_message`